runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -331,6 +332,12 @@ public void onMenuEntryAdded(MenuEntryAdded event)
 		int itemId = event.getIdentifier();
 		String option = Text.removeTags(event.getOption()).toLowerCase();
 		String target = Text.removeTags(event.getTarget()).toLowerCase();
+		NPC npc  = client.getHintArrowNpc();
```

Comment:
Use a better name for this variable, maybe `hintArrowNpc`?

Reply:
done